tasks/copy_files_to_archive/bin/run_tests.sh

```diff
@@ -45,23 +45,7 @@ function check_rc () {
       exit 1
   fi
 }
-## copy the shared_recovery.py
-echo "INFO: Copying ORCA shared libraries ..."
-if [ -d orca_shared ]; then
-    rm -rf orca_shared
-fi
-
-mkdir orca_shared
-let return_code=$?
-check_rc $return_code "ERROR: Unable to create orca_shared directory."
 
-touch orca_shared/__init__.py
-let return_code=$?
-check_rc $return_code "ERROR: Unable to create [orca_shared/__init__.py] file"
-
-cp ../shared_libraries/recovery/shared_recovery.py orca_shared/
-let return_code=$?
-check_rc $return_code "ERROR: Unable to copy shared library [orca_shared/shared_recovery.py]"
 
```

Comment:
Should probably remove the `rm -rf orca_shared`

Reply:
yep, missed that one.